compositor_chromium/src/cef_ref.rs

```diff
@@ -64,31 +64,41 @@ impl<T: CefStruct> CefRefPtr<T> {
         Box::into_raw(Box::new(cef_ref)) as *mut T::CefType
     }
 
+    /// # Safety
+    ///
+    /// Can be only used on data that is not null and was initialzed by `CefRefData::new`
     pub unsafe fn from_cef<'a>(cef_data: *mut T::CefType) -> &'a mut Self {
         unsafe { &mut *(cef_data as *mut Self) }
     }
 
+    /// # Safety
+    ///
+    /// Can be only used on data that is not null and was initialzed by `CefRefData::new`
     unsafe fn from_base<'a>(base: *mut chromium_sys::cef_base_ref_counted_t) -> &'a mut Self {
         unsafe { &mut *(base as *mut Self) }
     }
 
     extern "C" fn add_ref(base: *mut chromium_sys::cef_base_ref_counted_t) {
         let self_ref = unsafe { Self::from_base(base) };
+        // `Ordering::Relaxed` - there is no need for operation synchronization
```

Comment:
```suggestion
        // `Ordering::Relaxed` - there is no need for operation synchronization, since nothing changes 
        // about the ownership of the data: we had non-exclusive access to the data before the increment 
        // and we still do after
```